src/app/containers/MediaPage/index.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+import React from 'react';
+
+const MediaPageContainer = ({ service, match }) => (
+  <React.Fragment>
```

Comment:
I think most parts of the application are using `<Fragment>`  rather than `<React.Fragment>`. 

```
import React, { Fragment } from 'react';

 const MediaPageContainer = ({ service, match }) => (
  <Fragment>
     ...
  </Fragment>
```